src/functions-and-arrays.js

```diff
@@ -52,14 +106,40 @@ const wordsUnique = [
   'bring'
 ];
 
-function uniquifyArray() {}
+function uniquifyArray(value, index, self) {
+    return self.indexOf(value) === index;
+  }
+  
+  let unique = words.filter(uniquifyArray);
 
 
 
 // Iteration #6: Find elements
 const wordsFind = ['machine', 'subset', 'trouble', 'starting', 'matter', 'eating', 'truth', 'disobedience'];
 
-function doesWordExist() {}
+function doesWordExist(arr, wrd) {
+  for (let i = 0; i < arr.length; i++) {
+    const found = arr[i];
+    
+    if ( found === wrd) {
+      return "true";
```

Comment:
I think you need to return a boolean, here you are returning the string "true". You want the boolean true

And likewise with the false